### DISCUSSION TITLE
Noi v0.2.0

### DISCUSSION THREAD

**Discussion by lencx** [lencx/Noi#4] [General]
> Noi = AI + Tools + Plugins + Community

## macOS

```bash
xattr -cr /Applications/Noi.app
```

|Preview|Preview|
|---|---|
|<img width="1234" alt="1" src="https://github.com/lencx/Noi/assets/16164244/7e8cc48a-3b9a-4731-a8c9-1d2cef395bd7">|<img width="1226" alt="2" src="https://github.com/lencx/Noi/assets/16164244/23c8b872-4c54-4d91-8b85-293f5b640ab0">|
|<img width="1232" alt="3" src="https://github.com/lencx/Noi/assets/16164244/60566721-9152-489d-95f2-b7ba24619374">|<img width="1227" alt="4" src="https://github.com/lencx/Noi/assets/16164244/3446b740-5cd8-4077-affe-8655beac9938">|

<hr /><em>This discussion was created from the release <a href='https://github.com/lencx/Noi/releases/tag/v0.2.0'>Noi v0.2.0</a>.</em>

**Comment by chibincode** [comment]
![CleanShot 2024-01-25 at 07 01 35](https://github.com/lencx/Noi/assets/60530811/5e788eb9-b312-414b-af6c-a34ed0826e92)
奇怪，我一直显示不了我的账户信息，但是在浏览器里是可以的。

**Reply by chibincode** [reply]
就是左侧

**Reply by lencx** [reply]
你发送消息有响应吗？
